statut.yaml

```diff
@@ -0,0 +1,632 @@
+ chapters:
+  - id: "1"
+    title: "Postanowienia ogólne"
+    paragraphs:
+      - id: "1"
+        content:
+          - text: "\\textbf{Stowarzyszenie Krakowski Klub Chombo}, zwane dalej \\textbf{Stowarzyszeniem}, działa na podstawie ustawy Prawo o stowarzyszeniach (Dz. U. 1989 Nr 20 poz. 104 wraz z późniejszymi zmianami) oraz niniejszego Statutu."
```

Comment:
I'm thinking, is it necessary to have distinction `text`/`list`? In YAML, these are already two distinct data structures, so maybe we can drop these tags and make the format less verbose?

Reply:
yeah, we can – I automatically went for a tagged union so that we add another cases but no other cases came up. xd ptal

Reply:
tbh now that I'm looking at this, it's hard to spot lists in this :eyes: At this point, I'm fine with either version.